packages/suite/src/support/messages.ts

```diff
@@ -8503,4 +8503,53 @@ export default defineMessages({
         id: 'TR_STAKE_CLAIM',
         defaultMessage: 'Claim',
     },
+    TR_STAKE_STAKED_AMOUNT: {
+        id: 'TR_STAKE_STAKED_AMOUNT',
+        defaultMessage: 'Staked amount',
+    },
+    TR_STAKE_APY: {
+        id: 'TR_STAKE_APY',
+        defaultMessage: 'Annual Percentage Yield',
+    },
+    TR_STAKE_DAYS_TO: {
+        id: 'TR_STAKE_DAYS_TO',
+        defaultMessage: '{days} days',
+    },
+    TR_STAKE_NEXT_PAYOUYT: {
+        id: 'TR_STAKE_NEXT_PAYOUYT',
+        defaultMessage: 'Next reward payout • Approx. {fiatAmount}',
+    },
+    TR_STAKE_STAKE_MORE: {
+        id: 'TR_STAKE_STAKE_MORE',
+        defaultMessage: 'Stake more',
+    },
+    TR_STAKE_UNSTAKE_TO_CLAIM: {
+        id: 'TR_STAKE_UNSTAKE_TO_CLAIM',
+        defaultMessage: 'Unstake to claim',
+    },
+    TR_STAKE_ETH_REWARDS_EARN_APY: {
+        id: 'TR_STAKE_ETH_REWARDS_EARN_APY',
+        defaultMessage:
+            'Your ETH rewards also earn the APY rate. Keep your funds staked or add more to increase your rewards.',
```

Comment:
```suggestion
            'Your {symbol} rewards also earn the APY rate. Keep your funds staked or add more to increase your rewards.',
```

To be correct in testnets

Reply:
https://github.com/everstake/trezor-suite/pull/5/commits/25028a4faf3c04394eda3e5a7fee23435adbf3ef